### DISCUSSION TITLE
How to always use latest version

### DISCUSSION THREAD

**Discussion by N6REJ** [community/community#25772] [Actions]
<p>I’m a bit confused on how to use the “Latest” version of actions.<br>
I tried 2 different actions, both of which have the “Latest” tag but when using <span class="mention">@Latest</span> the workflow fails saying it can’t find it.</p>
<pre><code class="lang-auto">        name: Check GitHub Pages status
        uses: crazy-max/ghaction-github-status@v2.2.0
        with:
          pages_threshold: major_outage
      -
        name: Deploy
        if: success() &amp;&amp; github.event_name != 'pull_request'
        uses: crazy-max/ghaction-github-pages@v2.5.0
</code></pre>

**Comment by airtower-luna** [comment, marked as answer]
<p>I checked the repository of the latter action, and “Latest” is not a tag there. I assume you mean the “latest” mark in the releases section, that just marks, well, the latest version. :wink:</p>
<p>Theoretically you could use the default branch of the repository, but that could be unstable.</p>
<p>The best option is probably this: <a href="https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot" class="inline-onebox">Keeping your actions up to date with Dependabot - GitHub Docs</a></p>

**Comment by N6REJ** [comment]
<p>tyvm.  I’ve got dependabot so I’ll rely on it.  Might need to configure it better.<br>
now if I can just figure out wth the site won’t build <em>sigh</em></p>
